Assets/GLTF/Scripts/GLTFJsonExtensions.cs

```diff
@@ -149,9 +149,9 @@ public static Quaternion ReadAsQuaternion(this JsonReader reader)
 			return quat;
 		}
 
-		public static Dictionary<string, T> ReadAsDictionary<T>(this JsonReader reader, Func<T> deserializerFunc)
+		public static Dictionary<string, T> ReadAsDictionary<T>(this JsonReader reader, Func<T> deserializerFunc, bool skipStartObjectRead = false)
 		{
-			if (reader.Read() && reader.TokenType != JsonToken.StartObject)
+			if (!skipStartObjectRead && reader.Read() && reader.TokenType != JsonToken.StartObject)
```

Comment:
What do we use this for?

Reply:
Reading "dictionaries" (really, JSON objects) of nested objects didn't work due to two places expecting to call the first read. This broke the reading of the Targets array.

Reply:
I would have expected it to be used in this PR if it was also added in this PR.  Am I missing something?

Reply:
Oh, it was hidden in the diff!  I see it now.

Reply:
Not sure why it's hidden, but it's definitely used.